pom.xml

```diff
@@ -267,22 +268,84 @@
     </plugins>
   </build>
 
-  <repositories>
-    <repository>
-      <id>central</id>
-      <layout>default</layout>
-      <url>https://repo1.maven.org/maven2</url>
-    </repository>
-    <repository>
-      <id>bintray-streamnative-maven</id>
-      <name>bintray</name>
-      <url>https://dl.bintray.com/streamnative/maven</url>
-    </repository>
```

Comment:
Dropping this repository means we can't rely on our streamnative's pulsar as before. Is this accepted?

Reply:
Our future releases will be released in the maven central warehouse, and there is no need to rely on the bintray warehouse.